src/stack.jl

```diff
@@ -31,29 +24,69 @@ end
 Stack{T}() where {T} = Stack(Deque{T}())
 Stack{T}(blksize::Integer) where {T} = Stack(Deque{T}(blksize))
 
+"""
+    isempty(s::Stack)
+
+Check if stack `s` is empty.
+"""
 Base.isempty(s::Stack) = isempty(s.store)
+
+
+"""
+    isempty(s::Stack)
+
+Return the number of elements in stack `s`.
+"""
 Base.length(s::Stack) = length(s.store)
-Base.eltype(::Type{Stack{T}}) where T = T
+
+
+"""
+    eltype(::Type{Stack{T}}) where {T}
+
+Return the type of the elements in the stack.
+"""
+Base.eltype(::Type{Stack{T}}) where {T} = T
 
 """
     first(s::Stack)
 
-Get the top item from the stack. Sometimes called peek.
+Get the top item from stack `s`. Also know as "peak".
 """
 Base.first(s::Stack) = last(s.store)
+
 Base.last(s::Stack) = first(s.store)
 
+"""
+    push!(s::Stack, x)
+
+Insert new element `x` in top of stack `s`.
+"""
 function Base.push!(s::Stack, x)
     push!(s.store, x)
     return s
 end
 
+"""
+   pop!(s::Stack) 
+
+Remove and return the top element from stack `s`.
+"""
 Base.pop!(s::Stack) = pop!(s.store)
 
+"""
+    empty!(s::Stack)
+
+Remove all elements from stack `s`.
+"""
```

Comment:
```suggestion
```

Per method dostrings are not needed when they add no value over the per function ones.